src/data/finset/basic.lean

```diff
@@ -1278,6 +1278,9 @@ range_succ
 
 theorem range_mono : monotone range := λ _ _, range_subset.2
 
+lemma mem_range_succ_iff {a b : ℕ} : a ∈ finset.range b.succ ↔ a ≤ b :=
```

Comment:
Should this be a simp lemma?

Reply:
If you make it a simp lemma then a ton of stuff breaks. In another branch I tried to get it working, but in the end I gave up. The issue is that a < b.succ <-> a <= b isn't a `simp` lemma. Anyway, it was harder than I expected. Should I persevere? I think the branch was `mem-range-succ-iff`.

Reply:
I think we can defer this to a later PR.